File: lista 15questoes/15 questoes em java/att01.java
class Carro {
    String marca;
    String modelo;
    int ano;

    Carro(String marca, String modelo, int ano) {
        this.marca = marca;
        this.modelo = modelo;
        this.ano = ano;
    }

    void mostrarInformacao() {
        System.out.println("Marca: " + marca + ", Modelo: " + modelo + ", Ano: " + ano);
    }
}

class Main {
    public static void main(String[] args) {
        Carro carro1 = new Carro("Mitsubishi", "Lancer", 2014);
        Carro carro2 = new Carro("Honda", "Civic", 2017);
        carro1.mostrarInformacao();
        carro2.mostrarInformacao();
    }
}

File: lista 15questoes/15questoes em golang/att01.go
package main

import "fmt"

type Carro struct {
    marca  string
    modelo string
    ano    int
}

func (c Carro) mostrarInformacao() {
    fmt.Printf("Marca: %s, Modelo: %s, Ano: %d\n", c.marca, c.modelo, c.ano)
}

func main() {
    carro1 := Carro{"Mitsubishi", "Lancer", 2014}
    carro2 := Carro{"Honda", "Civic", 2017}
    carro1.mostrarInformacao()
    carro2.mostrarInformacao()
}